### ISSUE TITLE
Improve accessibility for settings dialog

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplates#208]
Currently the SmartTemplates settings dialog cannot be opened without clicking the expand button, there should be a an accessible way provided to expand the dialog with keyboard only:

![image](https://user-images.githubusercontent.com/6314674/183293686-f55bd921-1e69-4c56-85d9-d15e40cdcef6.png)

Also the tabs to the right of "Variables" cannot be accessed using the cursor key anymore:

![image](https://user-images.githubusercontent.com/6314674/183293633-9b3496f8-7e00-4fcd-9b14-0dfa6660a965.png)

this is probably a deprecation in the user XUL language but this problem may be there for a long time.

**Comment by RealRaven2000** [comment]
Here is a preview version that adds some keyboard navigation capabilities to the settings dialog:

[smartTemplate-fx-3.13pre86.zip](https://github.com/RealRaven2000/SmartTemplates/files/9281185/smartTemplate-fx-3.13pre86.zip)

- the toolbar beside the accounts dropdown can now be accessed using the [Tab] button
- you can navigate the visible buttons of that toolbar with cursor left and right keys. the last button expands the dialog
- all tabs (compose case tabs that switch between new / respond / forward; tabs in the expanded parts of the dialog including variables and licenses) can now be navigated using cursor left and right buttons

to install this version, you must download the zip file above, then install the file from Thunderbird Add-ons Manager using "Install Add-on from File". Don't extract the contents of the zip file.

**Comment by RealRaven2000** [comment]
For accessibility, let's rename "Advanced" to "Expand Dialog" (in German: "Dialog erweitern")
Also  it would be better for accessibility to set the focus on the first tab (Variables) after expanding the dialog, as this makes keyboard only navigation shorter.

Problems encountered: merely setting focus() using the JS routine doesn't appear to make some controls "audible" in narrator. Specifically, the tabs on the expanded version of the settings dialog (Variables, Global Settings, Advanced Settings, Examples, About ST, ST Licenses and Support) are not announced by Narrator in Windows 7 when I navigate them using the Cursor arrow right / left keys. I am leaving the version here so I can test it in Windows 10:

[smartTemplate-fx-3.13pre89.zip](https://github.com/RealRaven2000/SmartTemplates/files/9309486/smartTemplate-fx-3.13pre89.zip)

**Comment by RealRaven2000** [comment]
Implemented in 3.13 - released 21/08/2022
